### DISCUSSION TITLE
12h Time format chart

### DISCUSSION THREAD

**Discussion by nkamenar** [dbuezas/lovelace-plotly-graph-card#280] [Q&A] [answered]
How do you get the time on the x-axis to display in 12 hour format? Seems crazy there isn't a simple toggle for this somewhere but I can't find it and I have tried messing around and googling for hours and can't get anything to work.

**Comment by dbuezas** [comment, marked as answer]
I think you'll need to change the tickformat.
See here: https://plotly.com/javascript/tick-formatting/

**Reply by nkamenar** [reply]
Thank you sir, that is what I needed. For anyone looking for a specific example of how to achieve this here it is:
```
layout:
  xaxis:
    tickformat: '%I:%M %p'
```
